Tools/ApplicationDebugger/jacobi/README.md

```diff
@@ -0,0 +1,340 @@
+# `jacobi` Sample
+
+The 'jacobi' code sample is a small DPC++ code sample for exercising application debugging using
+Intel&reg; Distribution for GDB\*. It is highly recommended that you go
+through this sample *after* you familiarize yourself with the [array-transform](https://github.com/oneapi-src/oneAPI-samples/tree/bed8a563ace4a31d00a4f1055f65d680fa4fe0a8/Tools/ApplicationDebugger/array-transform)
+sample and the 
+[Get Started Guide](https://software.intel.com/en-us/get-started-with-debugging-dpcpp)
+for the debugger.
+
+This sample contains two versions of the same program. One works correctly, 
```

Comment:
overall this README is great.  one _minor_ suggestion would be to make clear here that by default, both programs are built.  The user does not need to do anything to select one over the other.

Reply:
The single "make" builds both targets, in VS the solution build also builds both project, "jacobi-bugged" is set as a startup project.